frontend/src/components/authentication/AuthenticationTabs.jsx

```diff
@@ -56,7 +46,7 @@ class AuthenticationTabs extends Component {
     ];
     return (
       <div>
-        {!this.state.isAuthenticated && (
+        {!this.props.authenticated && (
```

Comment:
Is this even needed? Don't you only show this component in the parent component if it's not authenticated?

Reply:
Good catch!
Removed in commit #https://github.com/code-for-canada/project-thundercat/pull/279/commits/12c9ae3ed5d12e9bf8c96fb54c37dbb53c22d124